Lib/random.py

```diff
@@ -492,7 +492,11 @@ def choices(self, population, weights=None, *, cum_weights=None, k=1):
     ## -------------------- real-valued distributions  -------------------
 
     def uniform(self, a, b):
-        "Get a random number in the range [a, b) or [a, b] depending on rounding."
+        """Get a random number in the range [a, b) or [a, b] depending on rounding.
+
+        E[X] = (a + b) / 2
```

Comment:
Should we note that this is the mean, for users that might be unfamilliar with the expectation operator?

```suggestion
        (mean): E[X] = (a + b) / 2
```